### PR TITLE
Butler 12 name

### PR DESCRIPTION
**Pull request by labbydev**
Addresses [BUTLER-12](https://palantir.atlassian.net/browse/BUTLER-12) which came out of Issue #15 

## Testing Instructions

Run `npm install palantirnet/butler#BUTLER-12-name` to install butler as a project dependency
Observe that it installs instead of erroring out

### CODE REVIEW

**Comment by gleroux02** [comment]
Reviewing

File: README.md [hunk 1]
```diff
@@ -8,15 +8,16 @@ The beginnings of a Sculpin-based style guide creation tool.
 ## Add Butler to a project
 1.   If the project does not already have npm dependencies run `npm init` to create a `package.json` file
 
-1.   Add Butler as a dependency by running `npm install --save git://github.com/palantirnet/butler.git#remove-ruby`
+1.   Add Butler as a dependency by running `npm install --save palantirnet/butler`
 
-## Getting Started with Butler
 1.  From inside the Butler root, run `npm install`
```
Comment (gleroux02) [inline README.md]: I don't think we need to run npm install inside of butler anymore since is being installed already.

Comment (labbydev) [inline README.md]: But it's not installing butler's dependencies I don't think. Can you test this?

Comment (gleroux02) [inline README.md]: Just confirmed that It does install it's dependencies.

Comment (labbydev) [inline README.md]: Awesome, I'll delete this

**Comment by labbydev** [comment]
Fixing

**Comment by labbydev** [comment]
Should be good for review again @gleroux02

**Pull request merged**